gallery/psgallery/psgallery_deploy_to_azure_automation_requireLicenseAcceptance.md

```diff
@@ -0,0 +1,22 @@
+---
+ms.date:  2017-06-12
+contributor:  JKeithB
+ms.topic:  conceptual
+keywords:  gallery,powershell,cmdlet,psgallery
+title:  psgallery_deploy_to_azure_automation
+---
+
+Require License Acceptance on Deploy to Azure Automation
+===========================
+
+If the module being deployed to Azure Automation requires license acceptance, portal UI will show a disclaimer saying By clicking OK, you are accepting license terms.
```

Comment:
>By clicking OK, you are accepting license terms [](start = 118, length = 47)

This part should be in single or double quotes. And also please include whole message as we display in UI.

Reply:
Done.

---
In reply to: [138457371](https://github.com/PowerShell/PowerShell-Docs/pull/1620#discussion_r138457371) [](ancestors = 138457371)